clickhouse_connect/driver/insert.py

```diff
@@ -198,3 +199,12 @@ def _convert_numpy(self, np_array):
                 data[ix] = data[ix].tolist()
         self.column_oriented = True
         return data
+
+    def start_column(self, name: str):
```

Comment:
This gets called during the insert to tell us what the current column being inserted is, so I stored its name in case we run into an error for that column